packages/sdk/src/utils/index.ts

```diff
@@ -38,8 +38,34 @@ export function createTransaction(
   return bitcoin.payments[type]({ pubkey: key, network: networkObj });
 }
 
-export function hdNodeToChild(node: BIP32Interface, formatType: AddressFormats = "legacy", index = 0) {
-  const fullDerivationPath = DERIVATION_PATHS_WITHOUT_INDEX[formatType] + index;
+export function getDerivationPath(formatType: AddressFormats, accountIndex = 0, index = 0) {
```

Comment:
rename as per [BIP44 specification](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels)

```suggestion
export function getDerivationPath(formatType: AddressFormats, account = 0, addressIndex = 0) {
```